### PR TITLE
Stricter LoqusDB management in institute settings

### LINKED ISSUE #2352: Normal users shouldn't be able to change loqus settings on institutes settings
I see 2 main issues there:

- Any user can modify the Loqus settings for an institute
- The field accepts any string, so if an unprivileged user writes "blabla" the whole institute tries to connect to a blabla loqus that doesn't exist with either the app crashing of not finding any occurrence for the variant, which are both bad!

image

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality:
- Allow only admin users to modify the Loqus instance in use for the institute
- Transform the loqus textfield of this form into a select with only existing loqus instance ids as possible values

fix #2352 

**How to test - local scout demo**:
1. Create a new user that has not an admin role: 
```
scout --demo load user -i cust000 -u "Diana Prince" -m wonderwoman@dc.com
```
2. Log in with this user and see the difference between master  branch and this branch (The loqus field should not be visible on the bottom of the institute settings in this branch)
3. Log out and login with Clark Kent (that is an admin) and see the difference with master branch (on master you should have a textfield accepting any value and on this branch a select showing only one value --> default)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready for review

File: scout/server/blueprints/institutes/forms.py [hunk 1]
```diff
@@ -72,7 +73,9 @@ class InstituteForm(FlaskForm):
         "Available patient cohorts", validators=[validators.Optional()]
     )
     institutes = NonValidatingSelectMultipleField("Institutes to share cases with", choices=[])
-    loqusdb_id = TextField("LoqusDB id", validators=[validators.Optional()])
+
+    loqus_instances = [instance["id"] for instance in loqusdb.loqusdb_settings]
+    loqusdb_id = SelectField("LoqusDB id", [validators.Optional()], choices=loqus_instances)
```
Comment (dnil) [inline scout/server/blueprints/institutes/forms.py:78]: 👍

File: scout/server/blueprints/institutes/templates/overview/utils.html [hunk 2]
```diff
@@ -88,23 +88,24 @@ <h5 class="header">General Institute Settings</h5>
...
-          </div>
-          <div class="form-row mt-3 d-flex justify-content-left" id="Custom_LoqusDB">
-            <div class="col-md-2 align-self-center" >
-              {{ form.loqusdb_id.label(class="control-label", data_toggle="tooltip", data_placement="top", title="LoqudDB id", placeholder="Search HPO.." ) }}
-            </div>
-            <div class="col-md-6 align-self-center" >
-              {{ form.loqusdb_id(class='form-control')}}
-            </div>
-            <div class="col-md-6 align-self-center" ></div>
+            {% if "admin" in current_user.roles %}
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/utils.html:99]: :+1:

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/institutes/templates/overview/utils.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/utils.html:99]: Merging, thanks @dnil !

**Pull request merged**